docs/_posts/2018-07-14-42.2.4-release.md

```diff
@@ -0,0 +1,54 @@
+---
+title: PostgreSQL JDBC Driver 42.2.4 Released
+date: 2018-07-14 19:02:04 +0300
+categories:
+  - new_release
+version: 42.2.4
+---
+**Notable changes**
+
+### Changed
+- PreparedStatement.setNull(int parameterIndex, int t, String typeName) no longer ignores the typeName
+argument if it is not null [PR 1160](https://github.com/pgjdbc/pgjdbc/pull/1160)
+
+### Fixed
+- Fix treatment of SQL_TSI_YEAR, SQL_TSI_WEEK, SQL_TSI_MINUTE [PR 1250](https://github.com/pgjdbc/pgjdbc/pull/1250)
+
+
+<!--more-->
+
+**Commits by author**
+
+Christian Kotzbauer (1):
+
+* Fixed typo in CHANGELOG.md [PR 1249](https://github.com/pgjdbc/pgjdbc/pull/1249) [b20df919](https://github.com/pgjdbc/pgjdbc/commit/b20df919a45435804b17f77da3a80aedd6675e15)
+
+Dave Cramer (3):
+
+* use the correct date for the release [985b63b7](https://github.com/pgjdbc/pgjdbc/commit/985b63b7228f3ef34159b68c1a4192c749994ef3)
+* also fix spelling of error [b0162c0a](https://github.com/pgjdbc/pgjdbc/commit/b0162c0a44075586a666a9d431b2230e2568a973)
+* fix: setNull for types not in java.sql.Types (e.g. uuid) [PR 1160](https://github.com/pgjdbc/pgjdbc/pull/1160) [6287c954](https://github.com/pgjdbc/pgjdbc/commit/6287c9547880b5cdfc596cf932354511a4e310cb)
+
+Jan Van den Bergh (1):
+
+* fix: map integrity constraint violation to XA_RBINTEGRITY instead of XAER_RMFAIL [PR 1175](https://github.com/pgjdbc/pgjdbc/pull/1175) [f2d1352c](https://github.com/pgjdbc/pgjdbc/commit/f2d1352c2b3ea98492beb6127cd6d95039a0b92f)
```

Comment:
this should be in the fixed section above I think

Reply:
done